src/lib.rs

```diff
@@ -138,6 +138,30 @@ impl RenetChannelsExt for RepliconChannels {
     }
 }
 
+/// External trait for [`ClientId`] to provide convenient conversion into renet ClientId.
+pub trait RenetClientIdExt {
+    /// Returns server channel configs that can be used to create [`ConnectionConfig`](renet::ConnectionConfig).
+    fn as_renet_client_id(&self) -> renet::ClientId;
```

Comment:
Maybe call it `to_renet` and `to_replicon`? This way this:
```rust
let renet_client_id = client_id.as_renet_client_id();
```
will look like this:
```rust
let renet_client_id = client_id.to_renet();
```

Reply:
Do you reckon the other `as_client_id` should change as well?

Reply:
Yes, I would change it into `to_replicon`.